backend/src/main/scala/com/softwaremill/adopttapir/template/ProjectTemplate.scala

```diff
@@ -0,0 +1,88 @@
+package com.softwaremill.adopttapir.template
+
+import com.softwaremill.adopttapir.starter.{StarterConfig, StarterDetails}
+import com.softwaremill.adopttapir.template.sbt.Dependency.{PluginDependency, ScalaDependency}
+import os.RelPath
+
+case class FileTemplate(
+    relativePath: RelPath,
+    content: String
+)
+
+class ProjectTemplate(config: StarterConfig) {
+
+  def getBuildSbt(starterDetails: StarterDetails): FileTemplate = {
+    val content = txt
+      .sbtBuild(
+        starterDetails.projectName,
+        starterDetails.groupId,
+        config.scalaVersion,
+        httpDependencies = List(
+          ScalaDependency("com.softwaremill.sttp.tapir", "tapir-core", "0.17.19"),
```

Comment:
we should use latest tapir version, which in turn should be configurable in the app (moving forward we'll want an automated build that redeploys this app when a new tapir version is released)

Reply:
Added to requirement that it will be provided in query parameter